src/aggregate/aggregate_request.c

```diff
@@ -1229,7 +1230,7 @@ static ResultProcessor *buildGroupRP(PLN_GroupStep *gstp, RLookup *srclookup,
     if (!ff) {
       // No such reducer!
       Grouper_Free(grp);
-      QueryError_SetErrorFmt(err, QUERY_ENOREDUCER, "No such reducer: %s", pr->name);
+      QueryError_SetWithoutUserDataFmt(err, QUERY_ENOREDUCER, "No such reducer: %s", pr->name);
```

Comment:
An illegal reducer - why is it different from an illegal format, e.g., in
```QERR_MKBADARGS_FMT(status, "FORMAT", " %s is not supported", format);```
Which is calling  QueryError_Set**With**UserDataFmt